resources/views/pages/ticket/[ticket].blade.php

```diff
@@ -0,0 +1,423 @@
+                        @if ($existingUnlock->unlocker_id)
+                            <span>Manually unlocked by {!! userAvatar(User::firstWhere('id', $existingUnlock->unlocker_id), icon:false) !!} at {{ getNiceDate($unlockedAt->unix()) }}</span>
+                        @else
+                            {{ $ticket->reporter->User }} earned this achievement at 
+                            {{ getNiceDate($unlockedAt->unix()) }}
+                            @if ($unlockedAt > $ticket->ReportedAt)
+                                (after the report)
+                            @else
+                                (before the report)
+                            @endif
+                        @endif
+                    @else
+                        <div class="flex w-full justify-between border-embed-highlight items-center">
+                            {{ $ticket->reporter->User }} did not earn this achievement
+                            @if ($permissions >= Permissions::Moderator)
+                                <script>
+                                    function AwardManually(hardcore) {
+                                        showStatusMessage('Awarding...');
+
+                                        $.post('/request/user/award-achievement.php', {
+                                            user: '{{ $ticket->reporter->User }}',
+                                            achievement: {{ $ticket->achievement->id }},
+                                            hardcore: hardcore
+                                        })
+                                        .done(function () {
+                                            location.reload();
+                                        });
+                                    }
+                                </script>
+                                <div class="flex items-center gap-x-3">
+                                    <button class="btn" onclick="AwardManually(0)">Award Softcore</button>
+                                    @if ($ticket->Hardcore)
+                                    <button class="btn" onclick="AwardManually(1)">Award Hardcore</button>
+                                    @endif
+                                </div>
+                            @endif
+                        </div>
+                    @endif
+                </div>
+            </div>
+        </div>
+    @endif
+
+    <div class="mt-2">
+        <div class="flex w-full justify-between border-embed-highlight items-center">
+            <p role="heading" aria-level="2" class="mb-0.5 text-2xs font-bold">
+                Comments
+            </p>
+            @if ($permissions >= Permissions::Developer)
+                <div class="flex flex-col gap-y-1">
+                    <a class="btn py-2 block transition-transform lg:active:scale-[97%]" href="{!! $contactReporterUrl !!}">
+                        Contact the reporter
```

Comment:
Recommend changing this to "Message the reporter" so it's consistent with the canonical term we use for the feature: "Messages".

Reply:
done.